setup.py

```diff
@@ -44,6 +44,9 @@
     include_package_data=True,
     install_requires=REQUIRES,
     python_requires=">=3.5",
+    extras_require={
+        'tests': ['pytest'],
+    },
```

Comment:
`pip install .[tests]` pulls in the optional tests dependencies (making it convenient for development).

This also means having dev dependencies listed explicitly in `setup.py`